### ISSUE TITLE
No apk built for release 1.9.9 ?

### ISSUE THREAD

**Issue by Saroumane** [connectbot/connectbot#1281]
Hello, I like to download apk directly from their github page, without any 3rd party involved.
It seems the latest connectbot apks are for version 1.9.8, there is only source available for 1.9.9.
Is it normal ?
Thanks.

**Comment by mattiklock** [comment]
I have this question too. The last release Obtainium grabs is 1.9.98, even though 1.9.9 is marked Latest.

**Comment by rusty-snake** [comment]
But 1.9.9 has no apk attached and "Fallback to older releases" is default in Obtainium.

**Comment by mattiklock** [comment]
Yes, Obtainium falls back to 1.9.8 just fine. We'd just like to be able to grab 1.9.9 without Google Play :)

**Comment by kruton** [comment]
I've updated the build to be more robust on the upload. It appears to be working for the last few months.
